file-format-types.rdf

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0"?>
+<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+  xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+  xmlns:owl="http://www.w3.org/2002/07/owl#"
+  xmlns:skos="http://www.w3.org/2004/02/skos/core#"
+  xmlns:udfrs="http://www.udfr.org/onto#">
+  <owl:Ontology rdf:about="http://pcdm.org/file-format-types#">
+    <rdfs:comment xml:lang="en">PCDM File Format Genre ontology.</rdfs:comment>
+    <rdfs:label xml:lang="en">PCDM File Format Genres</rdfs:label>
+  </owl:Ontology>
+  <udfrs:GenreFacetType rdf:ID="Document">
```

Comment:
Probably, though hopefully with some supporting criteria? Also need to make sure that Udfr Document aligns correctly, and verify that ebucore:Application isn't that concept also.